addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -14,7 +14,7 @@
 
 
 # Custom patches to perform the WSDL requests.
-EUSKADI_CIPHERS = f"{DEFAULT_CIPHERS}:!DH"
+EUSKADI_CIPHERS = f"DEFAULT:!DH"
```

Comment:
There seems to be something wrong here. Either remove the `f` or maybe you're missing some variable expansion?

```suggestion
EUSKADI_CIPHERS = "DEFAULT:!DH"
```